### PR TITLE
Expanse: Bug Fix - update jwt refresh to adhere to verify configuration

### PR DESCRIPTION
**Pull request by HilaryLynch-expanse**
Please ensure your pull request (PR) adheres to the following guidelines:

- Please refer to our contributing documentation for any questions on submitting a pull request, link: [Contribution Guide](https://github.com/splunk-soar-connectors/.github/blob/main/.github/CONTRIBUTING.md)

#### Please check if your PR fulfills the following requirements:
- [ ✅ ] Testing of all the changes has been performed (for bug fixes / features)
- [ ✅ ] The readme.html has been reviewed and added / updated if needed (for bug fixes / features)
- [ ✅ ] Use the following format for the PR description: `<App Name>: <PR Type> - <PR Description>`
- [ ✅ ] Provide release notes as part of the PR submission which describe high level points about the changes for the upcoming GA release.
- [ ✅ ] Verify all checks are passing.
- [ ✅ ] Do NOT use the `next` branch of the forked repo. Create separate feature branch for raising the PR.
- [ ✅ ] Do NOT submit updates to dependencies unless it fixes an issue.
- [ ✅ ] Limit your pull request to one application and submit multiple pull requests if necessary.
#### Please check the type of change your PR introduces:
- [ ] New App
- [ ✅  ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Documentation
- [ ] Other (please describe): 

## Release Notes (REQUIRED)
- Update to the behavior for refreshing an Expander JWT to adhere to the configuration for verifying certs.
- Updates to formatting to adhere to SOAR guidelines.
- Update min phantom app version to adhere to automated checks.

## What is the current behavior? (OPTIONAL)
- Describe the current behavior that you are modifying.

## What is the new behavior? (OPTIONAL)
- Describe the behavior or changes that are being added by this PR.

## Other information (OPTIONAL)
- Any other information that is important to this PR such as screenshots of how the component looks before and after the change.

## Pay close attention to (OPTIONAL)
- Any specific code change or test case points which must be addressed/reviewed at the time of GA release.

## Screenshots (if relevant)

---
Thanks for contributing!

### CODE REVIEW

File: expanse_connector.py [hunk 1]
```diff
@@ -217,14 +217,16 @@ def _fetch_jwt(self, config):
...
             "User-Agent": EXPANSE_USER_AGENT,
             "Authorization": "Bearer {}".format(self._token),
             "Content-Type": JSON_CONTENT_TYPE,
         }
         r = requests.get("{}/api/v1/idtoken".format(
             self._base_url),
-            headers=headers
+            headers=headers,
+            verify=config.get('verify_server_cert', False)
         )
```
[the remark this answers is not in the document]
Comment (HilaryLynch-expanse) [reply expanse_connector.py]: Updated

**Comment by HilaryLynch-expanse** [comment]
@arathore-crest Do we need to fix these tests before the PR will be merged? Thanks!

**Comment by arathore-crest** [comment]
@HilaryLynch-expanse, Yes, we need to fix these static tests before merging the PR. I will resolve these failures. Thanks!

**Comment by HilaryLynch-expanse** [comment]
@arathore-crest Great, thank you! Let me know if you need anything else from us.

**Review by brian-phantom — APPROVED** [review]

**Review by mjiyani-crest — APPROVED** [review]

**Pull request merged**